### PR TITLE
[Assignment #2] Convert to a plugin

### PR DESCRIPTION
**Pull request by pruett**
Reference issue #43

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -1,30 +1,19 @@
...
-            description: 'Returns the version of the server',
-            handler: function (request, reply) {
-
-                return reply({ version: Package.version });
-            }
-        }
+        Hoek.assert(!err, err);
     });
 
     server.start(function (err) {
```
Comment (AdriVanHoudt) [inline lib/index.js]: you are starting your server without being sure the plugin is fully registered, take a look at it

File: lib/version.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+var Package = require('../package.json');
+
+exports.register = function (server, options, next) {
+    server.route({
```
Comment (AdriVanHoudt) [inline lib/version.js]: newline after function

File: lib/version.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+                return reply({ version: Package.version });
+            }
+        }
+    });
+
+    return next();
+};
+
+exports.register.attributes = { name: 'versionRoute' };
```
Comment (hueniverse) [inline lib/version.js]: Cleaner to put object on multiple lines.

**Comment by hueniverse** [comment]
See general comments on issue.

**Pull request closed without merging**